README.md

```diff
@@ -106,3 +106,11 @@ Getting Started on AWS
     --root-drive=hello-rootfs.ext4 \
     --kernel-opts="console=ttyS0 noapic reboot=k panic=1 pci=off nomodules rw"
   ```
+
+## Testing
```

Comment:
This file uses "the other style" for marking headings. Please keep it consistent.